catalog-rest-service/src/main/java/org/openmetadata/catalog/security/DefaultAuthorizer.java

```diff
@@ -52,7 +51,8 @@
   private UserRepository userRepository;
 
   private PolicyEvaluator policyEvaluator;
-  private static final String FIELDS_PARAM = "roles,teams";
+  private Fields fieldsTeams;
+  private Fields fieldsRolesAndTeams;
 
   @Override
   public void init(AuthorizerConfiguration config, Jdbi dbi) throws IOException {
```

Comment:
@sureshms doing it in Authorizer is not the right thing to populate the security context. By the time request reaches is here security context is already populated. We should add this in JWTFilter https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/java/org/openmetadata/catalog/security/JwtFilter.java
https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/java/org/openmetadata/catalog/security/JwtFilter.java#L87
https://github.com/open-metadata/OpenMetadata/blob/main/catalog-rest-service/src/main/java/org/openmetadata/catalog/security/JwtFilter.java#L107

Reply:
we will refactor the above in the PR of policy refactoring